mlflow/tracing/processor/mlflow.py

```diff
@@ -92,14 +93,16 @@ def _start_trace(self, span: OTelSpan) -> TraceInfo:
                 "To avoid performance and disambiguation issues, set the experiment for "
                 "your environment using `mlflow.set_experiment()` API."
             )
-        default_tags = resolve_tags()
-        default_tags.update({TRACE_SCHEMA_VERSION_KEY: str(TRACE_SCHEMA_VERSION)})
+        tags = resolve_tags()
```

Comment:
Just to confirm, this code change is just a pure rename?

Reply:
Yes 😆